nuxeo-platform-user-registration-core/src/main/java/org/nuxeo/ecm/user/registration/UserRegistrationComponent.java

```diff
@@ -165,7 +166,13 @@ public String submitRegistrationRequest(UserRegistrationInfo userInfo, Map<Strin
     public String submitRegistrationRequest(String configurationName, UserRegistrationInfo userInfo,
             DocumentRegistrationInfo docInfo, Map<String, Serializable> additionnalInfo,
             ValidationMethod validationMethod, boolean autoAccept, String principalName)
-                    throws UserRegistrationException {
+            throws UserRegistrationException {
+
+        // First check that we have the originating user for that request
+        if (StringUtils.isBlank((String) additionnalInfo.get(PARAM_ORIGINATING_USER))) {
```

Comment:
`isEmpty` is enough. We don't check for "blank" for values that don't come from user input.